quinn-proto/src/shared.rs

```diff
@@ -192,15 +193,38 @@ pub struct EndpointConfig {
     ///
     /// Must be persisted across restarts to be useful.
     pub reset_key: SigningKey,
+
+    // SigningKey does not implement Clone, but we'd like our builders to be cloneable
+    reset_key_bytes: [u8; 64],
```

Comment:
A private field breaks things (see CI). If we need this, I think we might as well do away with the `SigningKey` field entirely and instead construct it internally to the connection, which I've been wanting anyway so as to reduce the degree to which other crates are part of our interface (a major semver compat hazard).

Reply:
This is now solved, at least!